django/applications/catmaid/control/label.py

```diff
@@ -78,6 +78,25 @@ def labels_all(request, project_id=None):
     return HttpResponse(json.dumps(labels), content_type='application/json')
 
 @api_view(['GET'])
+@requires_user_role(UserRole.Browse)
+def get_label_stats(request, project_id=None):
+    """Get usage statistics of node labels.
+    """
```

Comment:
Need a response spec if this is exposed via `api_view`. I can handle this sometime though, since Swagger is finnicky.